optuna/integration/allennlp/_train.py

```diff
@@ -0,0 +1,406 @@
+"""
+This file is ported from AllenNLP original implementation. (Under Apache 2.0 license)
+https://github.com/allenai/allennlp/blob/main/allennlp/commands/train.py
+"""
+
+import logging
```

Comment:
How about using Optuna's logging module?

Reply:
44bb1b1 changes the logging module.